packages/shared/src/components/post/PostContentContainer.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classed from '../../lib/classed';
+
+interface PostContentContainerProps {
+  hasNavigation: boolean;
+  className?: string;
+  children: ReactNode;
+}
+
+const BodyContainer = classed('div', 'flex flex-col w-full pb-6');
+
+const PageBodyContainer = classed(
+  BodyContainer,
+  'm-auto w-full laptop:border-l laptop:border-theme-divider-tertiary',
+);
+
+function PostContentContainer({
+  hasNavigation,
+  className,
+  children,
+}: PostContentContainerProps): ReactElement {
+  const Wrapper = hasNavigation ? BodyContainer : PageBodyContainer;
+
+  return <Wrapper className={className}>{children}</Wrapper>;
```

Comment:
You can also use ConditionalWrapper, it's an existing component, but up to you :)

Reply:
I think this is different since the wrapper here changes not the content itself which the `ConditionalWrapper` handles.